handsontable/src/plugins/multiColumnSorting/multiColumnSorting.js

```diff
@@ -23,6 +23,41 @@ registerRootComparator(PLUGIN_KEY, rootComparator);
  * This plugin sorts the view by columns (but does not sort the data source!). To enable the plugin, set the
  * {@link Options#multiColumnSorting} property to the correct value (see the examples below).
```

Comment:
It seems that the `::: example` notation is currently not accepted by our ESLint configuration. '

Note that no other JSDocs in `handsontable/src` use `::: example`. All JSDocs comments use `@example` instead. 

What we can do:
1. work on chaning the ESLint configuration to allow interactive examples (`::: example`) in JSDoc
2. use `@example`, which will be non-intertactive (same as on the page https://handsontable.com/docs/api/column-sorting/)

---

Most importantly, @adrianszymanski89 are you changing the right page? This PR changes the API reference page https://handsontable.com/docs/api/multi-column-sorting/.

It is not clear for me from the issue https://github.com/handsontable/handsontable/issues/9686, but I think @krzysztofspilka wanted to update the spiritual successor to the page https://handsontable.com/docs/row-sorting/, which is https://handsontable.com/docs/row-sorting/.

Reply:
I'm with @warpech  on that it should be in the https://handsontable.com/docs/row-sorting/,  but I think ideally it would be in both places. In this way, the user sees all the options in the [primary article row sorting]( https://handsontable.com/docs/row-sorting/), but also thanks to the search engine will come to a specific example [MulticolumnSorting.](https://handsontable.com/docs/api/multi-column-sorting/)

Reply:
Of course, @kirszenbaum's solution in the first comment also works. I am fine with it if you want.

Reply:
@warpech 

Thanks for checking that out. From the issue, I understood that that's the correct place to add that example. However, it would make sense to have it in both places, as @aninde said. 

@krzysztofspilka are you ok with that solution? If we add that example here: https://handsontable.com/docs/row-sorting/#row-sorting, I think it would need some explanation, and probably, a separate section, as it would look weird just being there on its own.

Reply:
@krzysztofspilka Can I please have your opinion on this? I'd like to move on with this task.

Reply:
It turned out that, indeed, the change was in the wrong place. @krzysztofspilka has clarified the issue, and now everything should be correct.

Reply:
Ok, but now I don't understand why just 1 demo was randomly changed to use `multiColumnSorting` without explanation, while 4 others still use `columnSorting`. In my opinion, noone will see it, or if they see it, they will be confused. The page should be clearer about what examples show what functionality.